crates/swc/src/plugin.rs

```diff
@@ -165,50 +138,8 @@ impl RustPlugins {
     #[cfg(all(any(feature = "plugin"), target_arch = "wasm32"))]
     #[tracing::instrument(level = "info", skip_all)]
     fn apply_inner(&mut self, n: Program) -> Result<Program, anyhow::Error> {
-        use std::{path::PathBuf, sync::Arc};
-
-        use anyhow::Context;
-        use swc_common::{
-            collections::AHashMap, plugin::serialized::PluginSerializedBytes, FileName,
-        };
-        use swc_ecma_loader::resolve::Resolve;
-
-        let should_enable_comments_proxy = self.comments.is_some();
-
-        swc_plugin_proxy::COMMENTS.set(
-            &swc_plugin_proxy::HostCommentsStorage {
-                inner: self.comments.clone(),
-            },
-            || {
-                let program = swc_common::plugin::serialized::VersionedSerializable::new(n);
-                let mut serialized = PluginSerializedBytes::try_serialize(&program)?;
-
-                if let Some(plugins) = &mut self.plugins {
-                    for p in plugins.drain(..) {
-                        let mut transform_plugin_executor =
-                            swc_plugin_runner::create_plugin_transform_executor(
-                                &PathBuf::from(&p.0),
-                                &swc_plugin_runner::cache::PLUGIN_MODULE_CACHE,
-                                &self.source_map,
-                                &self.metadata_context,
-                                Some(p.1),
-                            )?;
-
-                        serialized = transform_plugin_executor
-                            .transform(
-                                &serialized,
-                                self.unresolved_mark,
-                                should_enable_comments_proxy,
-                            )
-                            .with_context(|| {
-                                format!("failed to invoke `{}` as js transform plugin", &p.0)
-                            })?;
-                    }
-                }
-
-                serialized_program.deserialize().map(|v| v.into_inner())
-            },
-        )
+        // [TODO]: unimplemented
```

Comment:
Is this fine?

Reply:
Yes, this execution path is for @swc/wasm only (https://github.com/swc-project/swc/pull/7408/files#diff-8987cc214ab0c7d63b833d31785ecb48d89d1bcc204d42508c26ecd72becc7b9L165) which never worked correctly, so taking out for now and revise once we have proper support.